### PR TITLE
Improves the README file

### PR DESCRIPTION
**Pull request by dotdotcommadot**
To Improve the README file, I have:
- Added the inline code-documentation that already existed to the README
- Created a clear structure

Comments are more than welcome!

### CODE REVIEW

**Comment by dotdotcommadot** [comment]
Will do!

File: README.md [hunk 1]
```diff
@@ -117,3 +53,886 @@ bool _areTwoListsEqual(List<int> a, List<int> b) {
...
+- `predicate(S accumulated, T value, int index)`
+- `S seed` (optional)
+
+##### Returns: Observable
+
+–––
+
+#### <a id="skip"></a> skip
+
+Skips the first count data events from this stream.
```
Comment (brianegan) [inline README.md]: Hrm, a bit tough as well. Perhaps (adapted from RxJS):

Skips a specified number of items in an observable sequence and then continues with the remaining items.

File: README.md [hunk 2]
```diff
@@ -117,3 +53,886 @@ bool _areTwoListsEqual(List<int> a, List<int> b) {
...
+##### Arguments
+- `T startValue`
+
+##### Returns: Observable
+
+–––
+
+#### <a id="startWithMany"></a> startWithMany
+
+Prepends a value to the source Observable.
```
Comment (brianegan) [inline README.md]: Prepends a list of items...

**Comment by dotdotcommadot** [comment]
Since the API is changing fast, I think it might be better to:
- Keep the README clean, remove all detailed API documentation and just keep the API overview in the README
- Generate Dart Docs
- Refer to the Dart Docs from the README once we're nearing version 1
- Take all valuable comments from @brianegan added to this pull request, and use them to update the Dart Docs

**Pull request merged**